new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -200,30 +235,100 @@ public void testLogLevelSearch() {
 
-    verifySearchResults(queryCriteria, regex, LOGSTASH_INDEX_KEY, true);
+    verifyCountsHitsInSearchResults(queryCriteria, regex, LOGSTASH_INDEX_KEY, true);
 
     logger.info("Query logs of level=INFO succeeded");
   }
 
   /**
-   * Use Elasticsearch Search APIs to query Operator log info. Verify that log hits for
-   * type=weblogic-operator are not empty
+   * Use Elasticsearch Search APIs to query Operator log info. Verify that log occurrence for
+   * type=weblogic-operator are not empty.
    */
   @Test
   @DisplayName("Use Elasticsearch Search APIs to query Operator log info and verify")
   public void testOperatorLogSearch() {
-    // Verify that log hits for Operator are not empty
+    // Verify that log occurrence for Operator are not empty
     String regex = ".*took\":(\\d+),.*hits\":\\{(.+)\\}";
     String queryCriteria = "/_search?q=type:weblogic-operator";
 
-    verifySearchResults(queryCriteria, regex, LOGSTASH_INDEX_KEY, false);
+    verifyCountsHitsInSearchResults(queryCriteria, regex, LOGSTASH_INDEX_KEY, false);
 
     logger.info("Query Operator log info succeeded");
   }
 
-  private static void createAndVerifyDomain() {
-    // get the pre-built image created by IntegrationTestWatcher
-    String miiImage = MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG;
+  /**
+   * Use Elasticsearch Search APIs to query WebLogic log info.
+   * Verify that WebLogic server status of "RUNNING" is found.
+   */
+  @Disabled("Disabled the test due to JIRA OWLS-83899")
+  @Test
+  @DisplayName("Use Elasticsearch Search APIs to query Operator log info and verify")
+  public void testWebLogicLogSearch() {
+    // Verify that the admin status of "RUNNING" is found in query return from Elasticsearch repository
+    verifyServerRunningInSearchResults(adminServerPodName);
+
+    // Verify that the ms status of "RUNNING" is found in query return from Elasticsearch repos
+    verifyServerRunningInSearchResults(managedServerPodPrefix + "1");
+
+    logger.info("Query Operator log for WebLogic server status info succeeded");
+  }
+
+  /**
+   * Use Elasticsearch Search APIs to query WebLogic log info pushed to Elasticsearch repository
+   * by WebLogic Logging Exporter. Verify that log occurrence for WebLogic servers are not empty.
+   */
+  @Test
+  @DisplayName("Use Elasticsearch Search APIs to query WebLogic log info in WLS server pod and verify")
+  public void testWlsLoggingExporter() throws Exception {
+    Map<String, String> wlsMap = verifyLoggingExporterReady(opNamespace, null, WEBLOGIC_INDEX_KEY);
+    // merge testVarMap and wlsMap
+    testVarMap.putAll(wlsMap);
+
+    // Verify that occurrence of log level = Notice are not empty
+    String regex = ".*took\":(\\d+),.*hits\":\\{(.+)\\}";
+    String queryCriteria = "/_search?q=level:Notice";
+    verifyCountsHitsInSearchResults(queryCriteria, regex, WEBLOGIC_INDEX_KEY, false);
+    // Verify that occurrence of loggerName = WebLogicServer are not empty
```

Comment:
leave a blank line before this line for better code readability

Reply:
done